src/babel-preset-adeira/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+# 1.2.0
+
+- Add support for new jsx transform
```

Comment:
Could you please be a bit more helpful when crafting the changelog and include _at least_ some link explaining what's going on? For example, this one: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Also, `jsx` -> `JSX`